INSTALL

```diff
@@ -45,13 +45,21 @@ WHAT DO YOU NEED ?
    - for Coqide, the Lablgtk development files, and the GTK libraries
      including gtksourceview, see INSTALL.ide for more details
 
+   Note that camlp5 and lablgtk should be properly registered with
+   findlib/ocamlfind as Coq's makefile will use it to locate the
+   libraries during the build.
+
    Opam (https://opam.ocaml.org/) is recommended to install ocaml and
    the corresponding packages.
 
    $ opam install ocamlfind camlp5 lablgtk-extras
 
    should get you a reasonable OCaml environment to compile Coq.
 
+   Advanced users may want to experiment with the OCaml Flambda
+   compiler as way to improve the performance of Coq, see the entry
+   -flambda-opts below for more details.
```

Comment:
My comment at https://github.com/coq/coq/pull/7150#discussion_r178812397 applies better to the present PR actually. Would you mind enhancing this paragraph for instance by giving the command to switch to a Flambda compiler using OPAM ?

Reply:
Done, thanks. No problem in rebasing the other PR.